apidef/oas/middleware.go

```diff
@@ -475,3 +478,76 @@ type Header struct {
 	Name  string `bson:"name" json:"name"`
 	Value string `bson:"value" json:"value"`
 }
+
+func (ps Paths) fillCache(ep apidef.ExtendedPathsSet) {
+	advanceCacheConfig := &ep.AdvanceCacheConfig
+
+	for _, advCache := range *advanceCacheConfig {
+		if ps[advCache.Path] != nil {
+			plugins := ps[advCache.Path].getMethod(advCache.Method)
+			plugins.fillCache(advCache)
+
+		}
+	}
+}
+
+func (p *Plugins) fillCache(cacheMeta apidef.CacheMeta) {
```

Comment:
We can get rid of this one if we apply above logic. That approach is used in other plugins too, it is good for consistency.